homeassistant/components/netatmo/sensor.py

```diff
@@ -59,8 +59,6 @@
     "sum_rain_1": ["Rain last hour", LENGTH_MILLIMETERS, "mdi:weather-rainy", None],
     "sum_rain_24": ["Rain today", LENGTH_MILLIMETERS, "mdi:weather-rainy", None],
     "battery_percent": ["Battery Percent", PERCENTAGE, None, DEVICE_CLASS_BATTERY],
-    "min_temp": ["Min Temp.", TEMP_CELSIUS, None, DEVICE_CLASS_TEMPERATURE],
-    "max_temp": ["Max Temp.", TEMP_CELSIUS, None, DEVICE_CLASS_TEMPERATURE],
     "windangle": ["Direction", None, "mdi:compass-outline", None],
     "windangle_value": ["Angle", DEGREE, "mdi:compass-outline", None],
```

Comment:
We need to go through this list and start marking a bunch of these sensors as default disabled (using the `entity_registry_enabled_default` property on the Entity). There are too many and some are not useful for most, like this one.

Reply:
> We need to go through this list and start marking a bunch of these sensors as default disabled (using the `entity_registry_enabled_default` property on the Entity). There are too many and some are not useful for most, like this one.

Looks like a lazy person answer to the problem here.